openshift-hack/test-kubernetes-e2e.sh

```diff
@@ -78,6 +79,9 @@ mkdir -p "${test_report_dir}"
 SERVER=
 SERVER="$( kubectl config view | grep server | head -n 1 | awk '{print $2}' )"
 
+OTE_PATH="${GOPATH}/bin/k8s-tests-ext"
```

Comment:
Failing in CI on:

```
/usr/bin/test-kubernetes-e2e.sh: line 82: GOPATH: unbound variable
```

Reply:
Thanks, I actually don't think we need the specific path now as it is available on the PATH. I have updated it.